app/src/main/java/org/schabi/newpipe/error/ErrorPanelHelper.kt

```diff
@@ -127,6 +132,7 @@ class ErrorPanelHelper(
             ) {
                 // show retry button only for content which is not unavailable or unsupported
                 errorRetryButton.isVisible = true
+                showAndSetOpenInBrowserButtonAction(errorInfo)
```

Comment:
I think it's better if you put this line outside the `if (errorInfo.throwable !is ContentNotAvailableException &&
                errorInfo.throwable !is ContentNotSupportedException
            )`, since it's ok to allow the user to go to the browser even when the content is unavailable or unsupported